.github/workflows/build.yml

```
@@ -0,0 +1,38 @@
+name: Go
+on: [push]
+jobs:
+  run:
```

stylistic, since the name of the overall workflow is related to "build"ing, consider the job name to be `build`, again, stylistics, not a big deal

```suggestion
  build:
```

---

done